docs/usage/quickstart.rst

```diff
@@ -159,49 +164,55 @@ first run)::
     Launching MetalK8s shell environment. Run 'exit' to quit.
     (metal-k8s) $
 
-Now we're all set to deploy a cluster::
+Deploy the Cluster
+------------------
+
+Run the following command to deploy the cluster::
+
+  (metal-k8s) $ ansible-playbook -i inventory/quickstart-cluster -b playbooks/deploy.yml
 
-    (metal-k8s) $ ansible-playbook -i inventory/quickstart-cluster -b playbooks/deploy.yml
+For a simple test deployment such as the present three-node cluster, this takes
+about a half hour. Actual deployment time will vary based on the size of the
+cluster and hardware and network performance.
 
-Grab a coffee and wait for deployment to end.
+Inspect the Cluster
+-------------------
+Deployment creates a file containing credentials to access the cluster
+(:file:`inventory/quickstart-cluster/artifacts/admin.conf`). Remaining in the
+virtual environment shell, export this location to give :program:`kubectl` and
+:program:`helm` the correct credentials to contact the cluster’s *kube-master*
+nodes::
 
-Inspecting the cluster
-----------------------
-Once deployment finished, a file containing credentials to access the cluster
-is created: :file:`inventory/quickstart-cluster/artifacts/admin.conf`. We can
-export this location in the shell such that the :program:`kubectl` and
-:program:`helm` tools know how to contact the cluster *kube-master* nodes, and
-authenticate properly::
+    (metal-k8s) $ export KUBECONFIG=`pwd`/admin.conf
 
-    (metal-k8s) $ export KUBECONFIG=`pwd`/inventory/quickstart-cluster/artifacts/admin.conf
+If your system can reach port 6443 on a master node, you can
```

Comment:
Not *a* master node, but the 'first' master node (where 'first' can in theory be any of them). Basically, the master node referred to in `admin.conf`.